src/agent/src/rpc.rs

```diff
@@ -919,6 +919,13 @@ impl protocols::agent_ttrpc::AgentService for agentService {
         _ctx: &ttrpc::TtrpcContext,
         req: protocols::agent::UpdateInterfaceRequest,
     ) -> ttrpc::Result<Interface> {
+        if req.interface.is_none() {
+            return Err(ttrpc::Error::RpcStatus(ttrpc::get_status(
+                ttrpc::Code::INTERNAL,
```

Comment:
Thanks @jodh-intel.

How about changing status code from `INTERNAL` to `INVALID_ARGUMENT`? And use a more explicit message like "empty interfaces/empty routes" ?

Reply:
Agreed - updated! :smile: